### PR TITLE
Added -_-

### PR DESCRIPTION
**Pull request by kamala-nayana**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -34,4 +36,4 @@
 
 linked_list.delete_last()
 linked_list.print_list()
-print("========================")
```
Comment (gulsntils) [inline main.py:37]: Why did you delete this line?

Comment (kamala-nayana) [reply main.py:37]: Sorry Sir

Comment (Niteshiya) [reply main.py]: because you told so :-3
